src/Sarif.Multitool/Rules/RuleResources.resx

```diff
@@ -344,7 +344,6 @@ Many tool use similar names for 'uriBaseId' symbols. We suggest 'REPOROOT' for t
     <value>Placeholder_SARIF2015_EnquoteDynamicMessageContent_FullDescription_Text</value>
   </data>
   <data name="SARIF2015_EnquoteDynamicMessageContent_Note_Default_Text" xml:space="preserve">
-    <value>Placeholder_SARIF2015_EnquoteDynamicMessageContent_Note_Default_Text</value>
->>>>>>> features/sarif-validation-rules
```

Comment:
Incomplete merge.

Reply:
Oh! It looks like _I'm_ the one that did the incomplete merge :-). Thanks for fixing.

---
In reply to: [447906622](https://github.com/microsoft/sarif-sdk/pull/1947#discussion_r447906622) [](ancestors = 447906622)